cromwell_tools/scripts/cromwell-tools

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python2.7
+
+import argparse
+import json
+from cromwell_tools import cromwell_tools
+
+
+def start_workflow(arguments):
+    user, password = cromwell_tools.harmonize_credentials(
+        arguments.secrets_file, arguments.username, arguments.password)
+    wdl_file = cromwell_tools.download(arguments.wdl_file)
+    inputs_json = cromwell_tools.download(arguments.inputs_json)
+    try:
+        dependencies_bytes = cromwell_tools.download(arguments.dependencies_json)
+        dependencies_json = json.loads(dependencies_bytes)
+        dependencies_zip = cromwell_tools.make_zip_in_memory(
+            {k: cromwell_tools.download(v) for k, v in dependencies_json.items()}
+        )
+    except AttributeError:
+        dependencies_zip = None
+    try:
+        options_json = cromwell_tools.download(arguments.options_json)
+    except AttributeError:
+        options_json = None
+    try:
+        inputs2_json = cromwell_tools.download(arguments.inputs2_json)
+    except AttributeError:
+        inputs2_json = None
+
+    response = cromwell_tools.start_workflow(
+        wdl_file=wdl_file,
+        inputs_file=inputs_json,
+        inputs_file2=inputs2_json,
+        zip_file=dependencies_zip,
+        options_file=options_json,
+        url=arguments.cromwell_url + '/api/workflows/v1',
+        user=user,
+        password=password,
+    )
+    print(response.json()['id'])
+
+
+def main(arguments=None):
+
+    if arguments.command == 'run':
+        start_workflow(arguments)
```

Comment:
We should be consistent - spell out all arguments here or don't do it for the other subcommands.

Reply:
Never mind, I see why you've split it out this way.

Reply:
I'll add a doc-string like I have in other packages explaining what arguments=None is doing there (as you realized, it's for testing).